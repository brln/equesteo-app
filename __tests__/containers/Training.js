import configureStore from 'redux-mock-store'
import React from "react"
import {fromJS, Map} from "immutable"
import renderer from "react-test-renderer"
import { Provider } from 'react-redux'
import thunk from "redux-thunk"

import TrainingContainer from '../../containers/Training'


const mockStore = configureStore([thunk])

test('render', () => {
  const store = mockStore(fromJS({
    pouchRecords: {
      horseUsers: {},
      users: {},
    },
    localState: {},
  }))

  const tree = renderer.create(
    <Provider store={store}>
      <TrainingContainer />
    </Provider>).toJSON()
  expect(tree).toMatchSnapshot()
})

