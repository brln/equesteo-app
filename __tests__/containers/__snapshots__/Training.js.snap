// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
>
  <View />
  <View />
  <View
    onLayout={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#a7a6ab",
            "elevation": 3,
            "flexDirection": "row",
            "height": 50,
            "justifyContent": "space-around",
            "shadowColor": undefined,
            "shadowOffset": undefined,
            "shadowOpacity": undefined,
            "shadowRadius": undefined,
          },
          Object {
            "backgroundColor": "#F8F8F8",
          },
          Object {},
        ]
      }
    >
      <View
        accessible={true}
        clickable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "flex": 1,
            "opacity": 1,
          }
        }
      >
        <View
          active={true}
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#F8F8F8",
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "center",
              },
              Object {
                "backgroundColor": "#5bc0eb",
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#007aff",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": "600",
                  "marginLeft": 7,
                  "marginRight": 7,
                },
                Object {
                  "fontSize": 15,
                },
                Object {
                  "color": "white",
                },
                Object {
                  "color": "white",
                },
              ]
            }
            uppercase={false}
          >
            Calendar
          </Text>
        </View>
      </View>
      <View
        accessible={true}
        clickable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "flex": 1,
            "opacity": 1,
          }
        }
      >
        <View
          active={false}
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#F8F8F8",
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "center",
              },
              Object {
                "backgroundColor": "#5bc0eb",
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#6b6b6b",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "marginLeft": 7,
                  "marginRight": 7,
                },
                Object {
                  "fontSize": 15,
                },
                undefined,
                Object {
                  "color": "#6b6b6b",
                },
              ]
            }
            uppercase={false}
          >
            Charts
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "white",
            "bottom": 0,
            "height": 4,
            "left": 0,
            "position": "absolute",
            "width": 375,
          }
        }
      />
    </View>
    <RCTScrollView
      alwaysBounceVertical={false}
      automaticallyAdjustContentInsets={false}
      contentOffset={
        Object {
          "x": 0,
        }
      }
      directionalLockEnabled={true}
      horizontal={true}
      keyboardDismissMode="on-drag"
      keyboardShouldPersistTaps="handled"
      onMomentumScrollBegin={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      pagingEnabled={true}
      scrollEnabled={false}
      scrollEventThrottle={16}
      scrollsToTop={false}
      showsHorizontalScrollIndicator={false}
    >
      <View>
        <View
          style={
            Object {
              "width": 750,
            }
          }
        >
          <View
            activeTabStyle={
              Object {
                "backgroundColor": "#5bc0eb",
              }
            }
            activeTextStyle={
              Object {
                "color": "white",
              }
            }
            heading="Calendar"
            style={
              Object {
                "backgroundColor": "#FFF",
                "flex": 1,
              }
            }
            tabStyle={
              Object {
                "backgroundColor": "#5bc0eb",
              }
            }
          >
            <View
              style={
                Object {
                  "flex": 1,
                }
              }
            >
              <View
                style={
                  Object {
                    "flex": 1,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "flex": 1,
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "flex": 1,
                        "flexDirection": "row",
                      }
                    }
                  >
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "justifyContent": "center",
                        }
                      }
                    >
                      <Text
                        style={
                          Object {
                            "color": "#C2C2C2",
                            "fontSize": 15,
                            "textAlign": "center",
                          }
                        }
                      >
                        M
                      </Text>
                    </View>
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "justifyContent": "center",
                        }
                      }
                    >
                      <Text
                        style={
                          Object {
                            "color": "#C2C2C2",
                            "fontSize": 15,
                            "textAlign": "center",
                          }
                        }
                      >
                        T
                      </Text>
                    </View>
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "justifyContent": "center",
                        }
                      }
                    >
                      <Text
                        style={
                          Object {
                            "color": "#C2C2C2",
                            "fontSize": 15,
                            "textAlign": "center",
                          }
                        }
                      >
                        W
                      </Text>
                    </View>
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "justifyContent": "center",
                        }
                      }
                    >
                      <Text
                        style={
                          Object {
                            "color": "#C2C2C2",
                            "fontSize": 15,
                            "textAlign": "center",
                          }
                        }
                      >
                        R
                      </Text>
                    </View>
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "justifyContent": "center",
                        }
                      }
                    >
                      <Text
                        style={
                          Object {
                            "color": "#C2C2C2",
                            "fontSize": 15,
                            "textAlign": "center",
                          }
                        }
                      >
                        F
                      </Text>
                    </View>
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "justifyContent": "center",
                        }
                      }
                    >
                      <Text
                        style={
                          Object {
                            "color": "#C2C2C2",
                            "fontSize": 15,
                            "textAlign": "center",
                          }
                        }
                      >
                        Sa
                      </Text>
                    </View>
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "justifyContent": "center",
                        }
                      }
                    >
                      <Text
                        style={
                          Object {
                            "color": "#C2C2C2",
                            "fontSize": 15,
                            "textAlign": "center",
                          }
                        }
                      >
                        Su
                      </Text>
                    </View>
                  </View>
                </View>
                <View
                  style={
                    Object {
                      "flex": 12,
                    }
                  }
                >
                  <RCTScrollView>
                    <View>
                      <RCTScrollView
                        data={Array []}
                        disableVirtualization={false}
                        getItem={[Function]}
                        getItemCount={[Function]}
                        horizontal={false}
                        initialNumToRender={10}
                        keyExtractor={[Function]}
                        maxToRenderPerBatch={10}
                        numColumns={1}
                        onContentSizeChange={[Function]}
                        onEndReachedThreshold={2}
                        onLayout={[Function]}
                        onMomentumScrollEnd={[Function]}
                        onScroll={[Function]}
                        onScrollBeginDrag={[Function]}
                        onScrollEndDrag={[Function]}
                        removeClippedSubviews={false}
                        renderItem={[Function]}
                        scrollEventThrottle={50}
                        stickyHeaderIndices={Array []}
                        updateCellsBatchingPeriod={50}
                        viewabilityConfigCallbackPairs={Array []}
                        windowSize={21}
                      >
                        <View />
                      </RCTScrollView>
                    </View>
                  </RCTScrollView>
                </View>
              </View>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "width": 750,
            }
          }
        >
          <View
            heading="Charts"
          />
        </View>
      </View>
    </RCTScrollView>
  </View>
</View>
`;
